Assignment2/bgDani-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,51 @@
+{
+    "name":"Daniel Berrocal Gonzalez",
+    "hometown":"Parla",
+    "age":"22",
+    "alma mater":"Universidad Politecnica de Madrid",
+    "parents name":"Miguel y Teresa",
+    "Asignaturas":{
+        "name1":"Semantic Web, Linked Data and Knowledge Graphs",
+        "name2":"Ingenieria del Software 2",
+        "teacher1":"Oscar Corcho",
+        "techer2":"Tomas San Feliu",
+        "description1":"Proporcionar la base teorica y practica sobre los fundamentos implicados en la generacion y publicacion de vocabularios y datos enlazados en la Web",
+        "description2":"Aplicar los aspectos de gestion al desarrollo software."
+    }
+}
+
+{
+    "@context":{
+        "desc":"http://www.example2.es/personaldescription/",
+        "rdfs":"http://www.w3.org/2000/01/rdf-schema#",
+        "sub":"http://www.example2.es/personaldescription/asignaturas#",
+        "name":"rdfs:label",
+        "hometown":"rdfs:label",
+        "age":"rdfs:label",
+        "alma":"rdfs:label",
+        "parents":"rdfs:label",
+        "name1":"rdfs:label",
+        "name2":"rdfs:label",
+        "teacher1":"rdfs:label",
+        "teacher2":"rdfs:label",
+        "description1":"rdfs:label",
+        "description2":"rdfs:label",
+    },
+    "@id":"http://www.example2.es/personaldescription/me",
+    "@type":"desc:name",
+    "name":"Daniel Berrocal Gonzalez",
+    "hometown":"Parla",
+    "age":"22",
+    "alma":"Universidad Politecnica de Madrid",
+    "parents":"Miguel y Teresa",
+    "Asignaturas":{
+        "@id":"http://www.example2.es/personaldescription/asignaturas",
+        "@type":"sub:listsubject",
+        "name1":"Semantic Web, Linked Data and Knowledge Graphs",
+        "name2":"Ingenieria del Software 2",
+        "teacher1":"Oscar Corcho",
+        "techer2":"Tomas San Feliu",
+        "description1":"Proporcionar la base teorica y practica sobre los fundamentos implicados en la generacion y publicacion de vocabularios y datos enlazados en la Web",
+        "description2":"Aplicar los aspectos de gestion al desarrollo software."
```

Comment:
To define two Subjects you might use an array and define inside it each Subject as an object -> [{subject1 with atrributes},{subject2 with attributes}]. Remember that a Subject is a data object that has a name, description, teachers; so, you can only represent these attributes instead of defining a new one for each Subject (e.g. name1, name2, etc.) Thus, try to define Subjects as an array and define their generic attributes. So, it will also allow you to define fewer attributes in @context